diff --git a/ui/v2.5/graphql/stash-tv.graphql b/ui/v2.5/graphql/stash-tv.graphql
new file mode 100644
index 00000000..08e195ed
--- /dev/null
+++ b/ui/v2.5/graphql/stash-tv.graphql
@@ -0,0 +1,128 @@
+query FindScenesForTv(
+  $filter: FindFilterType
+  $scene_filter: SceneFilterType
+  $ids: [ID!]
+) {
+  findScenes(
+    filter: $filter
+    scene_filter: $scene_filter
+    ids: $ids
+  ) {
+    count
+    filesize
+    duration
+    scenes {
+      ...TvSceneData
+    }
+  }
+}
+
+query FindSceneMarkersForTv(
+  $filter: FindFilterType
+  $scene_marker_filter: SceneMarkerFilterType
+  $ids: [ID!]
+) {
+  findSceneMarkers(
+    scene_marker_filter: $scene_marker_filter
+    filter: $filter
+    ids: $ids
+  ) {
+    count
+    scene_markers {
+      id
+      title
+      seconds
+      end_seconds
+      stream
+      primary_tag {
+        id
+        name
+      }
+      tags {
+        id
+        name
+      }
+      scene {
+        ...TvSceneData
+      }
+    }
+  }
+}
+
+query GetStashConfigForTv {
+  configuration {
+    plugins
+    ui
+  }
+  availableSavedSceneFilters: findSavedFilters(mode: SCENES) {
+    id
+    name
+  }
+  availableSavedMarkerFilters: findSavedFilters(mode: SCENE_MARKERS) {
+    id
+    name
+  }
+}
+
+fragment TvSceneData on Scene {
+  id
+  title
+  date
+  files {
+    duration
+    height
+    width
+    format
+  }
+  interactive
+  performers {
+    name
+    gender
+  }
+  paths {
+    funscript
+    vtt
+    screenshot
+    stream
+    caption
+    preview
+  }
+  studio {
+    name
+    parent_studio {
+      name
+    }
+  }
+  sceneStreams {
+    url
+    mime_type
+    label
+  }
+  captions {
+    language_code
+    caption_type
+  }
+  rating100
+  o_counter
+  o_history
+  organized
+  resume_time
+  scene_markers {
+    id
+    title
+    seconds
+    end_seconds
+    primary_tag {
+      id
+      name
+    }
+    tags {
+      id
+      name
+    }
+  }
+  tags {
+    id
+    name
+  }
+}
