diff --git a/ui/v2.5/graphql/data/scene-tv.graphql b/ui/v2.5/graphql/data/scene-tv.graphql
new file mode 100644
index 00000000..6952f442
--- /dev/null
+++ b/ui/v2.5/graphql/data/scene-tv.graphql
@@ -0,0 +1,50 @@
+fragment TvSceneData on Scene {
+  id
+  title
+  date
+  files {
+    duration
+    height
+    width
+    format
+  }
+  interactive
+  performers {
+    name
+    gender
+  }
+  paths {
+    funscript
+    vtt
+    screenshot
+    stream
+    caption
+  }
+  studio {
+    name
+    parent_studio {
+      name
+    }
+  }
+  sceneStreams {
+    url
+    mime_type
+    label
+  }
+  captions {
+    language_code
+    caption_type
+  }
+  resume_time
+  scene_markers {
+    title
+    seconds
+    end_seconds
+    primary_tag {
+      name
+    }
+    tags {
+      name
+    }
+  }
+}
diff --git a/ui/v2.5/graphql/queries/scene-tv.graphql b/ui/v2.5/graphql/queries/scene-tv.graphql
new file mode 100644
index 00000000..709a2843
--- /dev/null
+++ b/ui/v2.5/graphql/queries/scene-tv.graphql
@@ -0,0 +1,18 @@
+query FindScenesForTv(
+  $filter: FindFilterType
+  $scene_filter: SceneFilterType
+  $scene_ids: [Int!]
+) {
+  findScenes(
+    filter: $filter
+    scene_filter: $scene_filter
+    scene_ids: $scene_ids
+  ) {
+    count
+    filesize
+    duration
+    scenes {
+      ...TvSceneData
+    }
+  }
+}
